unittests/testdictmerge.py

```diff
@@ -10,6 +10,320 @@
 
 from archivist import dictmerge
 
```

Comment:
import each method explicitly - the trick to writing fast python is to minimise the size of the dict that tracks all variable that are in scope.

When importing a complete package like so:

      import os

the local scope wlll contain all members of the os package (such as os.remove for example) which python will have to search.

However when only using a small number of features of the os package import each one separately to keep the local namespace small viz:

      from os import remove

If remove appears too generic then rename like so:

       from os import remove as os_remove

and this will preserve readability

Another example:

      from json import dumps as json_dumps

This also has the addeda advantage that the imported stuff is more easily mocked in unittests if required.

Reply:
Thank you for this explanation, I have implemented the relevant changes to the imports.